### PR TITLE
Calculator

### PR DESCRIPTION
**Pull request by UhrevatovDima**

### CODE REVIEW

File: Main.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        Scanner in = new Scanner(System.in);
+        System.out.println("Operation - ");
+        System.out.println("Input first number");
+        int inMinus = in.nextInt();
+        System.out.println("Input second number");
+        int inMinus2 = in.nextInt();
+        return inMinus - inMinus2;
+    }
+
+    public static int multiplication(){
```
Comment (eugeneugrevatov) [inline Main.java]: `multiply` should be enough

File: Main.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        Scanner in = new Scanner(System.in);
+        System.out.println("Operation * ");
+        System.out.println("Input first number");
+        int inMultiplication = in.nextInt();
+        System.out.println("Input second number");
+        int inMultiplication2 = in.nextInt();
+        return inMultiplication * inMultiplication2;
+    }
+
+    public static int division(){
```
Comment (eugeneugrevatov) [inline Main.java]: `divide` sound better

File: Main.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        }
+        if(input.equals("/")){
+            System.out.println("Result " + Main.division());
+        }
+    }
+
+    public static int plus(){
+        Scanner in = new Scanner(System.in);
+        System.out.println("Operation + ");
+        System.out.println("Input first number");
```
Comment (eugeneugrevatov) [inline Main.java]: Do you see some pattern is those methods?
```
System.out.println("Input first number");
int num1 = in.nextInt();
System.out.println("Input second number");
int num2 = in.nextInt();
```

**Review by eugeneugrevatov — CHANGES_REQUESTED** [review]

File: Main.java (hunk 3 above)
Comment (UhrevatovDima) [reply Main.java]: Yes you are right

File: Main.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+    public static int plus(){
+        System.out.println("Operation + ");
+        int[] masPlus;
+        masPlus = input();
+        return masPlus[0] + masPlus[1];
+    }
+
+    public static int minus(){
+        Scanner in = new Scanner(System.in);
```
Comment (eugeneugrevatov) [inline Main.java]: never used - seem like leftovers

File: Main.java [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+            System.out.println("Result " + Main.multiply());
+        }
+        if(input.equals("/")){
+            System.out.println("Result " + Main.divide());
+        }
+    }
+
+    public static int[] input(){
+        Scanner in = new Scanner(System.in);
+        int[] mas = new int[2];
```
Comment (eugeneugrevatov) [inline Main.java]: give more descriptive name to variable, please

File: Main.java [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+        int[] masPlus;
+        masPlus = input();
+        return masPlus[0] + masPlus[1];
+    }
+
+    public static int minus(){
+        Scanner in = new Scanner(System.in);
+        System.out.println("Operation - ");
+        int[] masMinus;
+        masMinus = input();
```
Comment (eugeneugrevatov) [inline Main.java]: why don't init on declaration?

Comment (eugeneugrevatov) [reply Main.java]: and have better name

**Review by eugeneugrevatov — CHANGES_REQUESTED** [review]

**Pull request merged**